tests/conftest.py

```diff
@@ -354,6 +356,36 @@ def finalizer():
     return factory
 
 
+@pytest.fixture()
+def teardown_factory(request):
+    """
+    Tearing down a resource that was created during the test
+    """
+    instances = []
+
+    def factory(resource_obj):
+        """
+        Args:
+            resource_obj (OCS object): Object to teardown after the test
+
+        """
+        instances.append(resource_obj)
+
+    def finalizer():
+        """
+        Delete the Pod
```

Comment:
If I understand you are talking about generic cleanup of uncleaned objects? not just Pod right?

Reply:
Thanks!